src/test/java/com/conveyal/datatools/editor/controllers/api/EditorControllerTest.java

```diff
@@ -64,26 +71,53 @@ public static void setUp() throws Exception {
         LOG.info("{} setup completed in {} ms", EditorControllerTest.class.getSimpleName(), System.currentTimeMillis() - startTime);
     }
 
+    @AfterAll
+    public static void tearDown() {
+        project.delete();
+        feedSource.delete();
+    }
+
+    private static Stream<Arguments> createPatchTableTests() {
+        return Stream.of(
+            Arguments.of("route_short_name", "route", 6, "graphql/routes.txt", "routes"),
+            Arguments.of("organization_name", "attribution", 1, "graphql/attributions.txt", "attributions"),
+            Arguments.of("translation", "translation", 3, "graphql/translations.txt", "translations")
+        );
+    }
+
+    @ParameterizedTest
+    @MethodSource("createPatchTableTests")
+    public void canPatchTableTests(
+        String field,
+        String entity,
+        int expectedCount,
+        String graphQLQueryFile,
+        String table) throws IOException {
+        canPatchTable(field, entity, expectedCount, graphQLQueryFile, table);
```

Comment:
Non-blocking: It looks like `canPatchTable` is only used once, so it could be inlined.